akka-actor-typed/src/main/scala/akka/actor/typed/internal/receptionist/LocalReceptionist.scala

```diff
@@ -14,7 +14,7 @@ import akka.actor.typed.scaladsl.LoggerOps
 import akka.annotation.InternalApi
 import akka.util.TypedMultiMap
 
-import scala.collection.Set
+import scala.collection.immutable
```

Comment:
isn't this solved by removing the import altogether? Prefef.Set is immutable.Set

Reply:
Indeed, I think some leftover from the initial PR was confusing me.